docs/cost-optimisation.md

```diff
@@ -0,0 +1,148 @@
+# Cost Optimisation
+
+## Limit The Total Number Of Workflows And Pods
+
+> Suitable for all.
+
+A workflow (and for that matter, any Kuberenetes resource) will incure a cost as long as they exists in your cluster. 
+
+The workflow controller memory and CPU needs increase linearly with the number of pods and workflows you are currently running. 
+
+Limit the total number of workflows using:
+
+* Active Deadline - delete running workflow that do not complete in a set time
```

Comment:
It does not delete the workflow it terminates it, so the underlying workflow and pods will remain, just not in a running state.. Not sure activeDeadlineSeconds should be mentioned here, given that this is under "Limit the total number of workflows using:"

Reply:
done